backend/wmg/data/constants.py

```diff
@@ -14,4 +14,4 @@
     "EFO:0010010": "CEL-seq2",
 }
 
-WMG_PINNED_SCHEMA_VERSION = "5.0.0"
+WMG_PINNED_SCHEMA_VERSION = "5.1.0"
```

Comment:
Same as Cell Guide pinned version--after some more thought I think we should leave it at 5.0.0 and let Apps team handle when to bump it.